experiment/measurer.py

```diff
@@ -364,6 +367,22 @@ def run_cov_new_units(self):
         self.UNIT_BLACKLIST[self.benchmark] = (
             self.UNIT_BLACKLIST[self.benchmark].union(set(crashing_units)))
 
+    def generate_summary(self, cycle: int, summary_only=True):
+        """Transforms the .profdata file into json form."""
+        coverage_binary = coverage_utils.get_coverage_binary(self.benchmark)
+        result = coverage_utils.generate_json_summary(coverage_binary,
+                                                      self.profdata_file,
+                                                      self.cov_summary_file,
+                                                      summary_only=summary_only)
+        if result.retcode != 0:
```

Comment:
Personally would avoid the extra indent and do `if result.retcode == 0: return`

Reply:
Can fix in next CL.